datasets/wit/README.md

```diff
@@ -0,0 +1,346 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- found
+languages:
+- af
+- ar
+- ast
+- azb
+- be
+- bg
+- bn
+- br
+- ca
+- cs
+- cy
+- da
+- de
+- el
+- en
+- eo
+- es
+- et
+- eu
+- fa
+- fi
+- fr
+- fy
+- ga
+- gl
+- hr
+- hu
+- hy
+- id
+- it
+- iw
+- ja
+- ka
+- ko
+- la
+- lt
+- lv
+- mk
+- ml
+- ms
+- nl
+- nn
+- 'no'
+- pl
+- pt
+- ro
+- ru
+- sk
+- sl
+- sr
+- sv
+- th
+- tr
+- uk
+- ur
+- vi
+- vo
+- zh
+licenses:
+- cc-by-sa-3.0
```

Comment:
This is the license of Google's repo, no? For Papers With Code the dataset license is unknown

Reply:
Their repo mentions:
```
This data is available under the [Creative Commons Attribution-ShareAlike 3.0 Unported](https://github.com/google-research-datasets/wit/blob/main/LICENSE) license.
```